test/new-e2e/go.mod

```diff
@@ -44,7 +44,7 @@ require (
 	github.com/pulumiverse/pulumi-time/sdk v0.0.0-20231010123146-089d7304da13
 	github.com/samber/lo v1.39.0
 	github.com/sethvargo/go-retry v0.2.4
-	github.com/stretchr/testify v1.8.5-0.20231013065317-89920137cdfa
```

Comment:
This commit is included in v1.9.0 https://github.com/stretchr/testify/commit/89920137cdfa81f0d5e6e31fd2bf8a0d1aec8e8a